analyses/cms-open-data-ttbar/ttbar_analysis_pipeline.py

```diff
@@ -678,13 +679,13 @@ def get_query(source: ObjectStream) -> ObjectStream:
 all_histograms["hist"][120j::hist.rebin(2), "4j1b", :, "nominal"].stack("process")[::-1].plot(stack=True, histtype="fill", linewidth=1, edgecolor="grey")
 plt.legend(frameon=False)
 plt.title("$\geq$ 4 jets, 1 b-tag")
-plt.xlabel("$H_T$ [GeV]");
+plt.xlabel("$H_T$ [GeV]")
```

Comment:
These semicolons all have the purpose to suppress output that would otherwise be printed as far as I remember. Maybe there is a better way (we could assign `_ = ...` to achieve the same), but if not then I'd say it is better to keep them and disable the associated warning code.

Reply:
I see, another quirk of syncing the `.py` and the `.ipynb` versions I guess. I will just silence this type of linting error.

Reply:
Done